src/dbnode/storage/cleanup_test.go

```diff
@@ -710,3 +720,27 @@ func TestCleanupManagerCommitLogTimesMultiNS(t *testing.T) {
 	require.True(t, contains(filesToCleanup, time10))
 	require.True(t, contains(filesToCleanup, time20))
 }
+
+func TestCleanupManagerDeletesCorruptCommitLogFiles(t *testing.T) {
+	// TODO(rartoul): Re-enable this once https://github.com/m3db/m3/issues/1078
+	// is resolved.
```

Comment:
Can you add a note to the issue to "be sure to re-enable test `TestCleanupManagerDeletesCorruptCommitLogFiles` with the same PR that fixes this issue".

Reply:
sure